### PR TITLE
[wasm] Misc CI/test run improvements

### PR DESCRIPTION
**Pull request by radical**
- Update chrome+chromedriver for helix tests to latest stable (`96.0.4664.45`)
- Enable library tests to run on Windows, for manual runs

- Disable windows/AOT builds completely due to https://github.com/dotnet/runtime/issues/61721
- Disable failing WBT tests on windows due to https://github.com/dotnet/runtime/issues/61725
- Disable JS interop tests failing with new chrome - https://github.com/dotnet/runtime/issues/61945

### CODE REVIEW

**Review by lewing — APPROVED** [review]
The test exclusions and chrome bump look good, @steveisok can you take a look at the pipeline bits

File: src/libraries/sendtohelixhelp.proj [hunk 1]
```diff
@@ -364,15 +364,20 @@
       <WorkItemPrefix Condition="'$(Scenario)' == 'BuildWasmApps' and '$(TestUsingWorkloads)' == 'true'">Workloads-</WorkItemPrefix>
       <WorkItemPrefix Condition="'$(Scenario)' == 'BuildWasmApps' and '$(TestUsingWorkloads)' != 'true'">EMSDK-</WorkItemPrefix>
     </PropertyGroup>
+
+    <!-- Version number to revision number mapping from http://omahaproxy.appspot.com/ and find the closest one in
+         https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/
+
+         Eg. latest stable version is 96.0.4664.45 with revision 929512.
+         but the closest one available in the snapshosts is 929513
```
Comment (radekdoulik) [inline src/libraries/sendtohelixhelp.proj:372]:
```suggestion
         https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Linux_x64/
         and https://commondatastorage.googleapis.com/chromium-browser-snapshots/index.html?prefix=Win_x64/

         Eg. latest stable version is 96.0.4664.45 with revision 929512.
         but the closest one available in the snapshosts is 929513.
         Please make sure to check both platforms as sometime
         the same snapshot might not be available in one of them.
```

**Review by radekdoulik — APPROVED** [review]

**Review by steveisok — APPROVED** [review]
LGTM

**Pull request merged**